cime_config/testdefs/testmods_dirs/clm/ciso_cwd_hr/user_nl_clm

```diff
@@ -1,2 +1,2 @@
-paramfile = '$DIN_LOC_ROOT/lnd/clm2/paramdata/ctsm51_ciso_cwd_hr_params.c240105.nc'
+paramfile = '/glade/campaign/cgd/tss/people/oleson/modify_param/ctsm51_ciso_cwd_hr_params.c240208.nc'
```

Comment:
Move this into cesm inputdata as talked about above.

Reply:
Moved and rimported. Now I need to change the paths accordingly in these .xml and user_nl_clm files.

Reply:
@olyson maybe your GSWP3 spinup in #2403 should point to this parameter set from inputdata?  Not sure it will be critical since they are the same file...?

Reply:
The only difference between these files in the most recent version of CTSM:

ctsm51_params.c240207b.nc
ctsm51_ciso_cwd_hr_params.c240207b.nc

is "ceta" which is 450 in the ciso version and 358 in the normal version.  This parameter was changed from 450 to 358 in PR #250 .
I'm guessing that the ciso version didn't get changed at that point.
So I think I should be using the 358 version.  I think we decided we should get rid of the ciso version at some point.

Reply:
Oh, maybe I misunderstood your question.  You are suggesting I point to the non-iso rimported inputdata file, maybe to make sure it was imported correctly and readable, etc.?

Reply:
Turns out that was a good idea.  I can't read that file.  @slevis-lmwg can you change the permissions on /glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/paramdata/ctsm51_params.c240208.nc?

Reply:
Ok, done.

Reply:
I'm getting an error using this surface dataset with the deadveg branch (base is ctsm5.1.dev166):

/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_0.9x1.25_hist_1850_78pfts_c240216.nc

So maybe there IS a code modification that is not yet in the main code but is in the ctsm5.2_mksurfdata branch?  The error message is:

surfrd_get_data ERROR: sum of wt_lunit not    1.00000000000000       at nl=4
sum is:   0.000000000000000E+000

Something to do with PCT_OCEAN maybe?

Reply:
You're probably right that it's the PCT_OCEAN in the new fsurdat file.

I recommend this pre-PCT_OCEAN fsurdat file, which I think should be the same in other respects:
surfdata_0.9x1.25_hist_1850_78pfts_c231026.nc